### DISCUSSION TITLE
Automatic backup?

### DISCUSSION THREAD

**Discussion by abdessalaam** [mastodon/mastodon#21980] [Q&A]
Does anyone use any automatic backup solutions? For either mastodon instance itself, or if not possible, for the server that hosts it? Asking for recommendations.
I'm thinking of something aside from the usual backup offered by VM providers, so maybe ProxMox or UrBackup...

I run my instance in docker, so also maybe there are some automated container backup solutions?

Thanks!

**Comment by oculos** [comment]
Hi!
For now, I am following most of what is said here: 

https://docs.joinmastodon.org/admin/backups/

I have a script who dumps the database and save it on a remote location.
The same script saves the .env.production file.
Next I'll do is to save redis.

I could take snapshots of the vm and all, but I think this will do it for now.

**Comment by abdessalaam** [comment]
I am trying this: [docker-volume-backup](https://github.com/offen/docker-volume-backup)
